montage/rdb.py

```diff
@@ -3097,6 +3085,23 @@ def choose_eligible(eligible_users):
             'task_count_mean': mean(vote_count_map.values())}
 
 
+def get_source():
+    from utils import load_env_config
+
+    config = load_env_config()
+
+    config_source = config.get('source')
```

Comment:
`source` is a little vague here, can we call it `commons_source`?

An aside: `remote` and `local` are pretty bad names, too, I'll take the full blame. The proper meaning, maybe worth a comment: remote = toolforge util service (ideal name: `tools_api`), local = toolforge commons db replica (ideal name: `db_replica`).